packages/eventcatalog/pages/services/[name].tsx

```diff
@@ -36,9 +36,9 @@ function MermaidComponent({ title, service, charts }: { title?: string; service:
 
 const getComponents = (service: Service) => ({
   Admonition,
-  AsyncAPI: () => {
+  AsyncAPI: ({ url }: { url?: string }) => {
     if (!service.asyncAPISpec) return null;
-    return <AsyncApiSpec spec={service.asyncAPISpec} />;
+    return <AsyncApiSpec spec={service.asyncAPISpec} url={url} />;
```

Comment:
Not sure if I'm missing something, but is this valid? https://github.com/asyncapi/asyncapi-react#props

I can't see a URL prop for this component? Can you link some docs?

Reply:
Sorry think I found it now FetchingSchemaInterface, thanks looks good to me, will try it out....

Reply:
Sorry also helps if I read your PR description, sorry 😅